src/Agent/NewRelic/Agent/Extensions/NewRelic.Agent.Extensions/Parsing/SqlWrapperHelper.cs

```diff
@@ -20,7 +20,7 @@ public static class SqlWrapperHelper
     {
         private const string NullQueryParameterValue = "Null";
 
-        private static Regex _getDriverFromConnectionStringRegex = new Regex(@"DRIVER\=\{(.+?)\}");
+        private static Regex _getDriverFromConnectionStringRegex = new Regex(@"DRIVER\=\{(.+?)\}", RegexOptions.IgnoreCase);
```

Comment:
We might want to make this a compiled regex (`RegexOptions.Compiled`) too.

Reply:
Yeah, I meant to do that but forgot in all the confusion over the connect and iteration metrics.  Good catch!